queue_entry.py

```diff
@@ -553,7 +552,7 @@ def execute(self):
             # Create a centred postions of the current postion
             pos_dict = self.diffractometer_hwobj.getPositions()
             cpos = queue_model_objects.CentredPosition(pos_dict)
-            pos = shape_history.Point(None, cpos, None) #, True)
+            #pos = shape_history.Point(None, cpos, None) #, True)
```

Comment:
I followed Ivars’s suggestion and did the same change as that in Master branch on this part. I don’t have a BL or  machine to test it at our site. With the mockup it seems working.

I will look into this. Sorry about the problems…


From: Matias Guijarro [mailto:notifications@github.com]
Sent: Tuesday, February 02, 2016 9:52 AM
To: mxcube/HardwareObjects
Cc: Jie Nan
Subject: Re: [HardwareObjects] a few fixes of mockups (#9)


In queue_entry.py<https://github.com/mxcube/HardwareObjects/pull/9#discussion_r51539477>:

> @@ -553,7 +552,7 @@ def execute(self):

>              # Create a centred postions of the current postion

>              pos_dict = self.diffractometer_hwobj.getPositions()

>              cpos = queue_model_objects.CentredPosition(pos_dict)

> -            pos = shape_history.Point(None, cpos, None) #, True)

> +            #pos = shape_history.Point(None, cpos, None) #, True)

Sorry to come back on this...
I would be happy to merge the PR and to test it here (as we are in our machine day without users),
but it won't work: look at line 563 it does pos.qub_point etc.
Maybe we can just remove the lines 563 to 566 ?
Any hint would be appreciated.

—
Reply to this email directly or view it on GitHub<https://github.com/mxcube/HardwareObjects/pull/9/files#r51539477>.